UI/html/question.html

```diff
@@ -75,32 +75,33 @@ <h1>Subscribe to for news letter</h1>
             </li>
         </ul>
     </section>
+    <p id="deleteDoc"></p>
     <section id="all__blog">
         <div class="all__blog">
             <div class="item" id="single__article">
                 <!-- <div class="article"> -->
-                <h4><b>Benimana Eric</b></h4>
+                <!-- <h4><b>Benimana Eric</b></h4>
                 <p class="article__text">Prosciutto beef ribs strip steak, biltong chislic tri-tip shankle. Biltong drumstick alcatra, shankle leberkas brisket pork belly pork chop. Rump andouille pork chop, beef ribs fatback t-bone brisket shankle tur salami short ribs corned
                     beef tongue. <br>
                     <p id="like__icon"><b>bizip04@gmail.com</b>&nbsp;&nbsp;&nbsp;&nbsp;</p>
 
                     <p id="like__icon"><b>Delete</b></p>
 
```

Comment:
You should be removing unused code in your final delivery of a feature

Reply:
Yes, I will do that